spec/defines/foreman_plugin_spec.rb

```diff
@@ -29,10 +30,7 @@
     } end
 
     it 'should install the correct package with notify' do
```

Comment:
I suppose the plugin packages should restart the core app now in their postinst (iirc RPMs do this, DEBs not really)?